### DISCUSSION TITLE
How can I uninstall all plugins?

### DISCUSSION THREAD

**Discussion by yinghaoyu** [ayamir/nvimdots#1307] [Q&A] [answered]
I think it should be mentioned in wiki home page.

**Comment by charliie-dev** [comment]
u mean clean up all leftovers plugins? or literally all plugins?
if the former, u can use `:Lazy clean` to do it.

**Reply by yinghaoyu** [reply]
The second please.

**Comment by ayamir** [comment, marked as answer]
`rm -fr ~/.local/share/nvim/site/lazy/*`

**Reply by charliie-dev** [reply]
https://github.com/ayamir/nvimdots/wiki/Issues#how-to-uninstall-all-plugins

done
